app/src/main/java/com/teamagam/gimelgimel/app/map/actions/timeplay/TimeplayViewModel.java

```diff
@@ -247,6 +241,11 @@ private void stopPreviousSnapshotDisplay() {
   }
 
   private void startCurrentSnapshotDisplay(long newTimestamp) {
+    createSnapshotInteractor(newTimestamp);
+    mSnapshotInteractor.execute();
+  }
+
+  private void createSnapshotInteractor(long newTimestamp) {
```

Comment:
This should return an interactor and not assign one inside it.
So either change the name or change it to return an interactor that you'll assign to the field when calling it

Reply:
done